linux/mount.go

```diff
@@ -570,11 +570,16 @@ func MountDeviceWithFileSystem(devPath string, mountPoint string, options []stri
 	return mount, nil
 }
 
-func MountNFSShare(source string, targetPath string, options []string) error {
+func MountNFSShare(source string, targetPath string, options []string, nfsType string) error {
 	log.Tracef(">>>>> MountNFSShare called with source %s target %s", source, targetPath)
 	defer log.Tracef("<<<<< MountNFSShare")
 
-	args := []string{source, targetPath}
+	// default nfs version to 4.0
+	if nfsType == "" {
+		nfsType = "nfs4"
```

Comment:
Move this hard-coded string to constants.h

Reply:
done..